Mariuzzo.Do.DataAnnotations.Test/CedulaAttributeTest.cs

```diff
@@ -29,5 +31,78 @@ public void TestInvalidCedulas()
             Assert.IsFalse(attr.IsValid("\n"));
             Assert.IsFalse(attr.IsValid("\n \t"));
         }
+        [TestMethod]
+        public void TestValidCedulasExceptionsShouldReturnValidIfIgnoreExceptionsIsFalse()
+        {
+            var attr = new CedulaAttribute { IgnoreExceptions = false };
+            _validExceptions.ForEach(c => Assert.IsTrue(attr.IsValid(c)));
+        }
+        [TestMethod]
+        public void TestInvalidCedulaWhenIgnoreExceptionsIsFalse()
+        {
+            var attr = new CedulaAttribute();
+            _validExceptions.ForEach(c => Assert.IsFalse(attr.IsValid(c)));
+        }
+        #region Cédulas that do not fill the criteria of the algorithm
+        private readonly List<string> _validExceptions = new List<string>()
```

Comment:
No need to have the exact same array that appears on the RNCValidator class. I'd rather make the array on the implementation class a public static field there (as I mentioned on another comment) and use it here.